services/frontend-service/pkg/cmd/server.go

```diff
@@ -14,6 +14,24 @@ You should have received a copy of the GNU General Public License
 along with kuberpult.  If not, see <http://www.gnu.org/licenses/>.
 
 Copyright 2021 freiheit.com*/
+/*
```

Comment:
why does this keep getting added again?

Reply:
As far as I know, it is the ESLint formatting that always does that